### ISSUE TITLE
voice blending possible?

### ISSUE THREAD

**Issue by stlohrey** [Plachtaa/seed-vc#75]
Hey, thank you for the great work! Is it possible to do some kind of voice blending, similar to model blending in RVC? And if so, can you guide on how to do it?

Thank you!

**Comment by Plachtaa** [comment]
Thanks for your suggestion. 
Voice blending might be possible but not sure it works or not. We may conduct some experiments to find out.

**Comment by Mofa-Xingche** [comment]
Unless we can create new voices by blending multiple human voices with model merging, we will be haunted by copyright curses to the ends of hell, so this technology is absolutely necessary for practical use lol(>_<)

**Comment by TomSuen** [comment]
> Thanks for your suggestion. Voice blending might be possible but not sure it works or not. We may conduct some experiments to find out.

Hi, I have met with a problem that while the `source_audio` containing very emotional content, the clone result is not well, is there any optimization to do?@Plachtaa 

# source
https://drive.google.com/file/d/1ANP-pINInG_QSlN1HOjdV7Lx1vK79ax5/view?usp=sharing

# ref
https://drive.google.com/file/d/10cZxh6CQ47g6_aywh9be8olaTJlnLNHx/view?usp=sharing

# gen_result
https://drive.google.com/file/d/1qGlt5NZAVk-Dwp5rf2RmyE1ZBuMYDPS1/view?usp=sharing

**Comment by Plachtaa** [comment]
> > Thanks for your suggestion. Voice blending might be possible but not sure it works or not. We may conduct some experiments to find out.
> 
> Hi, I have met with a problem that once the `source_audio` containing very emotional content, the clone result is not well, is there any optimization to do?@Plachtaa
> 
> The `Diffusion Steps` is set to 100
> 
> # source
> https://drive.google.com/file/d/1ANP-pINInG_QSlN1HOjdV7Lx1vK79ax5/view?usp=sharing
> 
> # ref
> https://drive.google.com/file/d/10cZxh6CQ47g6_aywh9be8olaTJlnLNHx/view?usp=sharing
> 
> # gen_result
> https://drive.google.com/file/d/1qGlt5NZAVk-Dwp5rf2RmyE1ZBuMYDPS1/view?usp=sharing

Sounds fine to me. Do you mean the strong emotion was not reflected from the converted audio?

**Comment by TomSuen** [comment]
> Sounds fine to me. Do you mean the strong emotion was not reflected from the converted audio?
Yes, I would like to ask how to better preserve the mood of `source_audio` and clone the timbre of `ref_audio` well.

**Comment by Plachtaa** [comment]
> > Sounds fine to me. Do you mean the strong emotion was not reflected from the converted audio?
> 
> Yes, I would like to ask how to better preserve the emotion of `source_audio` and clone the timbre of `ref_audio` well.

Currently it is an issue, we will study more on that

**Comment by TomSuen** [comment]
> Currently it is an issue, we will study more on that

Look forward to your good news😀

**Comment by GUUser91** [comment]
@TomSuen 
The closest voice conversion app i could find that tries to preserve the emotion of the source audio while voice cloning the timbre of the reference audio is CosyVoice. I'm currently using CosyVoice 2.0, i hear that CosyVoice 3.0 is gonna do a better job at preserving the emotion of the source audio than CosyVoice 2.0. I made a thread that contains download links for a modded gradio CosyVoice app with batch voice conversion support
https://github.com/FunAudioLLM/CosyVoice/issues/1616
CosyVoice 2.0 example
Ref audio cleaned up with resemble enhance and Acon Digital DeVerberate 3
https://vocaroo.com/17RW2gGy0GKf
Source audio
https://vocaroo.com/1jUyQDCJApaI
gen result
https://vocaroo.com/1kGTbRyCYdpg
